### PR TITLE
feat: Use a separate configuration page

### PR DESCRIPTION
**Pull request by liuweiGL**
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

Replace the GlobalConfiguration view with the ManagementLink view，so that we can get a cleaner view:

image
image

### CODE REVIEW

File: src/main/resources/jenkins/plugins/publish_over_ssh/BapSshManagementLink/index.jelly [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler"
+  xmlns:l="/lib/layout" xmlns:f="/lib/form">
+  <l:layout permission="${app.ADMINISTER}" norefresh="true" title="钉钉配置页面">
```
Comment (gmcdonald) [inline src/main/resources/jenkins/plugins/publish_over_ssh/BapSshManagementLink/index.jelly:4]: Should title be a variable here?

**Comment by gmcdonald** [comment]
In general, nice idea, but I am not sure that this should belong on the main 'manage' page itself.
Would like thoughts from others, perhaps @MarkEWaite ?

**Comment by MarkEWaite** [comment]
> In general, nice idea, but I am not sure that this should belong on the main 'manage' page itself. Would like thoughts from others, perhaps @MarkEWaite ?

That's a good question that is probably better asked to @janfaracik and @timja and @uhafner .  They work much more with evolving the UI.

For the git plugin automatic cache maintenance project, we created a  separate page under "Manage Jenkins".  We did that because it needed to present cache status information in addition to accepting user input to configure the caching.  In this case, the information seems to be purely configuration information.  I'm OK with it being a separate page, but not sure if that's the direction that others want to go.

Moving the configuration to a separate page will probably require modifications of one or more acceptance tests that depend on the current location of the configuration information.  Before this is merged, I'd expect to see a description that confirms the acceptance tests have been run on the plugin and have been confirmed that they are passing.

If a separate page is added, then it should be categorized into one of the existing categories.  See https://github.com/jenkinsci/implied-labels-plugin/pull/4 and the specific commit https://github.com/jenkinsci/implied-labels-plugin/pull/4/commits/45c65d3c71526febff6ee865579238e5045017c7 for the categorization technique.

**Comment by timja** [comment]
I think this needs more justification. Doesn't seem like it needs to be on a management link from what I can see.

Yes we do want to breakup the configure system page to something more sensible but I don't think this is the way to do it

**Comment by uhafner** [comment]
I also think this should be a section in the global configuration. This is the place where all plugins put their data into.

**Comment by gmcdonald** [comment]
In addition, I think this might surprise and/or confuse people. They expect things in one place. Also if this were applied to publish-over-ssh then it would need to be applied for publish-over-[ftp|cifs|...] for consistency.

I agree with others that this should wait until until such a time as the UI team get together and work out a common approach to splitting up the config page.

**Comment by gmcdonald** [comment]
Thanks for the contribution, however as indicated in earlier comments, we will not be implementing this.

**Pull request closed without merging**